### ISSUE TITLE
Add warning about item's description contains newlines when running TWQueryTasks

### ISSUE THREAD

**Issue by yongli-abc** [huantrinh1802/m_taskwarrior_d.nvim#90]
Here's a test query:

<img width="545" alt="image" src="https://github.com/user-attachments/assets/7439a603-132e-4086-955a-9cb7bafc765c">

When I run `:TWQueryTasks`, I get the following error:
<img width="1056" alt="image" src="https://github.com/user-attachments/assets/2616980b-e6d7-488b-b465-0a8ad2753af5">

I added a print before [this line](https://github.com/huantrinh1802/m_taskwarrior_d.nvim/blob/d0cf7a7e882e308ab203d9973ce3147a328258f7/lua/m_taskwarrior_d/init.lua#L590), there's a newline string at the end.

Will send a PR to fix this.

**Comment by yongli-abc** [comment]
Yes, it took me quite a while to find the task which actually has a newline in the description. It's very obscure because it doesn't show in the report result, nor the edit view at all.

**Comment by huantrinh1802** [comment]
hey @yongli-abc, after reviewing the structure in general, the issue seems to be more widespread than this particular command.

I am still unsure if we would want to support description with newline (`\n`). If we do want to support it, how is the best way to handle it. For example, the approach via #91 is to escape the new line, which shows `\n` in the markdown file. This may not be the best option for readability. I would love to see your and others' opinions on this

**Comment by yongli-abc** [comment]
Maybe the support isn't really necessary. The newline was a mistake on my side and other tasks are all good. I also checked that Taskwiki has the same problem.

The fact that no one has raised this to you or Taskwiki likely shows how rare this is.

Happy to not fix this too :)

**Comment by huantrinh1802** [comment]
I think instead of supporting the newline. We can provide more insightful message. It could be something like this `The task 31de3 has a newline character. In order to render the task, please consider to remove the newline in the task's description`

**Comment by yongli-abc** [comment]
That sounds reasonable too!
